ansible/roles/bigbluebutton/molecule/default/molecule.yml

```diff
@@ -12,6 +12,7 @@ lint: |
 platforms:
   - name: ubuntu1604-instance
     box: generic/ubuntu1604
+    box_version: 3.0.1
```

Comment:
Why is this needed and where did the number come from?

I was only able to find these versions: https://app.vagrantup.com/ubuntu/boxes/xenial64

Reply:
Ah good catch this is not relevant any more as I was testing different versions. I fixed this in: https://github.com/PrivacyLx/devops/pull/55/commits/c559cdfc86214e921675a492bf48552cb8024fc4

For reference this is the image that we are using:
https://app.vagrantup.com/generic/boxes/ubuntu1604